creators/views.py

```diff
@@ -111,3 +111,22 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message" : "KEYERROR"}, status=401)
 
+class ProductListView(View):
+    @login_decorator
+    def post(self,request):
+        user = request.user
+
+        user_id = Creator.objects.get(user=user.pk)
+        products = Product.objects.filter(creator=user_id).prefetch_related('productthumbnailmedia_set')
```

Comment:
아래와 같이 줄여볼 수 있을 것 같습니다. filter 조건에 user로 걸어주세요~
```suggestion
        user = request.user
        products = Product.objects.filter(creator__user_id=user.pk).prefetch_related('productthumbnailmedia_set')
```